.eslintrc.js

```diff
@@ -1,4 +0,0 @@
-module.exports = {
-  root: true,
-  extends: '@react-native-community',
-};
```

Comment:
We're still planning to use an eslint-config though, right? is the community one not working well with TS?

Reply:
I intend to put linting back, I just had problems in my IDE with it and wanted to focus on TS first